alonerpg/tablesreader.py

```diff
@@ -23,17 +23,17 @@ def __init__(self, index, system, filename):
 def rt(chosen_table):
 
     table_name = chosen_table.filename.replace(".txt", "")
-    valid_string = re.search(r"^[a-zA-Z_\s]*$", table_name)
+    valid_string = re.search(r"^[a-zA-Z_\sñ0-9]*$", table_name)
 
     if not valid_string:
-        return Response(message="Invalid string", data=None)
+        return Response(message="Invalid string", data="")
 
     table_path= tables_path + "/" + chosen_table.system + "/" + table_name + ".txt"
 
     with open(table_path, encoding="utf-8", errors="ignore") as f:
         elements = [line.rstrip() for line in f]
 
-    return Response(message="Result:", data=random.choice(elements))
+    return Response(message="Result for '" + table_name + "':", data=random.choice(elements))
```

Comment:
Here i would use `f-strings`:
`f"Result for '{table_name}':"`

Reply:
Done